opentreemap/treemap/audit.py

```diff
@@ -29,7 +29,7 @@
 from treemap.util import (all_models_of_class, leaf_models_of_class,
                           to_object_name, safe_get_model_class)
 
-from treemap.lib.object_caches import (permissions,
+from treemap.lib.object_caches import (permissions as field_permissions,
```

Comment:
In `object_caches` we should rename `permissions` to `field_permissions` so this clarification would be used everywhere. Not necessarily as part of this PR.